requirements.txt

```diff
@@ -0,0 +1,14 @@
+coverage==7.6.1
+ete3==3.1.3
+jinja2==3.1.4
+jsonrpcbase==0.2.0
+numpy==2.0.1
+PyQt5==5.15.11
+pytest==8.3.2
+pytest-cov==5.0.0
+pytest-recording==0.13.2
+requests==2.32.3
+ruff==0.5.7
```

Comment:
there's only one dev requirement so I just included it in the main requirements file 🤷